preset/rabbitmq/preset.go

```diff
@@ -121,6 +137,148 @@ func (p *P) setDefaults() {
 	}
 }
 
+func (p *P) loadFiles() error {
+	if len(p.MessagesFiles) > 0 {
+		for _, fName := range p.MessagesFiles {
+			msgs, err := p.loadMessagesFromFile(fName)
+			if err != nil {
+				return fmt.Errorf("can't read messages from file '%s': %w", fName, err)
+			}
+
+			p.Messages = append(p.Messages, msgs...)
+		}
+	}
+
+	return nil
+}
+
+func declareQueues(ch *amqp.Channel, qs []string) error {
+	for _, queue := range qs {
+		_, err := ch.QueueDeclare(queue, false, false, false, false, nil)
+		if err != nil {
+			return fmt.Errorf("can't open queue '%s': %w", queue, err)
+		}
+	}
+
+	return nil
+}
+
+func (p *P) initf(ctx context.Context, c *gnomock.Container) (err error) {
+	conn, err := p.connect(c)
+	if err != nil {
+		return fmt.Errorf("can't connect to rabbitmq: %w", err)
+	}
+
+	defer func() {
+		closeErr := conn.Close()
+		if err == nil && closeErr != nil {
+			err = closeErr
+		}
+	}()
+
+	if err := p.loadFiles(); err != nil {
+		return err
```

Comment:
```suggestion
		return fmt.Errorf("can't read messages files: %w", err)
```

Reply:
There is already a similar message being returned from `loadFiles` since I've just copied the logic already written into a separate function to fix the linter error. Do you think there should be this additional message?

Reply:
I like to always add a message, but it's up to you here. I agree that with or without it the error is very clear.